addons/mail/models/mail_ice_server.py

```diff
@@ -45,7 +45,7 @@ def _get_ice_servers(self):
         :return: List of dict, each of which representing a stun or turn server,
                 formatted as expected by the specifications of RTCConfiguration.iceServers
         """
-        if self.env['ir.config_parameter'].get_param('mail.use_twilio_rtc_servers'):
+        if self.env['ir.config_parameter'].sudo().get_param('mail.use_twilio_rtc_servers'):
             (account_sid, auth_token) = self._get_twilio_credentials()
```

Comment:
you will need to add 2 sudo in this method as well

Reply:
done